modules/template/services/ExportInstanceService.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\custom_pages\modules\template\services;
+
+use humhub\modules\custom_pages\modules\template\elements\BaseElementContent;
+use humhub\modules\custom_pages\modules\template\elements\ContainerElement;
+use humhub\modules\custom_pages\modules\template\elements\ContainerItem;
+use humhub\modules\custom_pages\modules\template\models\TemplateInstance;
+use Yii;
+use yii\web\Response;
+
+class ExportInstanceService
```

Comment:
Maybe rename to `ExportTemplateInstanceService` and `ExportTemplateService`? Not sure it's useful?

Reply:
@luke- Yes, I thought about to rename to `ExportTemplateService` and `ImportTemplateService`, but Calendar module uses this class name [here](https://github.com/humhub/calendar/pull/545/files#diff-724326d4c9977d3f53ba0f053c7d857dc8b8ef4f2d61b2b377791b4ec560720dR43) and now I see the PR is not merged yet, so yes I can rename it without problems.

Reply:
Renamed in the commit https://github.com/humhub/custom-pages/pull/407/commits/767203e94a9a481917c49da8527f42c58268c909 and for Calendar module - https://github.com/humhub/calendar/pull/545/commits/f236648077411f4cf744f4df46fd0a883a3bc08c